site/js/components/ee.js

```diff
@@ -0,0 +1,227 @@
+ */
+
+/**
+ * Bootstrap the M100 Console Quiz
+ */
+export class M100 {
+  constructor() {
+    this.curPrompt = this._getCurrentPrompt() || 0;
+    this.quizStart = this._getQuizStart();
+
+    this.text = {};
+    this.text.intro = {
+      headline: 'Ace our quiz for a surprise.',
+      body: ["Head on a short scavenger hunt to celebrate Chrome OS's 100th release with us. All four answers can be found here on ChromeOS.dev; get them right for a fun surprise.", "Let's go!"],
+    };
+    this.text.prompts = [
+      {
+        question: 'What year did Chrome OS and the first Chromebooks launch?',
+        hint: 'We wrote a blog post about it.',
+        responseCorrect: 'Nice!',
+        responseIncorrect: "Let's try that again.",
+      },
+      {
+        question: 'How many millions of students and educators use Chromebooks?',
+        hint: 'Check the homepage.',
+        responseCorrect: 'Nice!',
+        responseIncorrect: 'Not quite.',
+      },
+      {
+        question: "What's the codename for Linux on Chrome OS?",
+        hint: 'We mention it on the Linux page.',
+        responseCorrect: 'Nice!',
+        responseIncorrect: 'Hmm, try again.',
+      },
+      {
+        question: 'How many weeks are there between Chrome OS releases?',
+        hint: 'Refer to Chromium Dash.',
+        responseCorrect: 'Nice!',
+        responseIncorrect: "You're almost there!",
+      },
+    ];
+    this.text.answers = ['2011', '50', 'Crostini', '4'];
+    this.text.reward = {
+      headline: 'Huzzah!',
+      body: ["You're officially a Chrome OS insider. We've got more exciting things coming your way for our 100th software release. To stay in the know, sign up for our newsletter, and keep exploring the all-new ChromeOS.dev to help you build your next big idea.", "Thank you for all of your support, we couldn't have reached this milestone without you!", 'The Chrome OS team'],
+    };
+
+    if (!this.quizStart) {
+      console.warn('%c%s', 'font-weight: bold;', 'm100.start()');
+    } else {
+      this._promptQuestion(this.curPrompt);
+    }
+  }
+
+  /**
+   * Get started
+   * @return {void}
+   */
+  start() {
+    if (!this.quizStart) {
+      this._setQuizStart(true);
+      console.info('%c%s', 'font-weight: bold; background: #174ea6; color: white; padding: .25em;', this.text.intro.headline);
+      for (const iterator of this.text.intro.body) {
+        console.info(iterator);
+      }
+    }
+    console.warn('Hint: to answer a question use: %c%s', 'font-family: monospace; font-weight: bold;', 'm100.answer(value);');
+    this._promptQuestion(this.curPrompt);
+  }
+
+  /**
+   * Answer
+   * @param {string} value - Submit an answer to a prompted question
+   * @return {void}
+   */
+  answer(value) {
+    if (!this.quizStart) {
+      console.error('Hmmm... what are you trying to answer?');
+    } else {
+      this._checkAnswer(this.curPrompt, value);
+    }
+  }
+
+  /**
+   * Start over
+   * @return {void}
+   */
+  clear() {
```

Comment:
```suggestion
  restart() {
```

Clear to me reads as wipe a screen, but not start over. I think this is more obviously "restart from the beginning"

Reply:
agreed +1 - this was originally just clearing vars for testing, but I find might be useful for the user to start over.